nvdaHelper/vbufBase/storage.cpp

```diff
@@ -321,6 +321,17 @@ bool VBufStorage_fieldNode_t::addAttribute(const std::wstring& name, const std::
 	return true;
 }
 
+bool VBufStorage_fieldNode_t::getAttribute(const std::wstring& name, std::wstring* value) {

```

Comment:
Don't take `std::wstring* value` as a pointer. Taking by reference would be more appropriate as it implies can't be `nullptr`. Better would be to return a `std::optional`.